src/Workspaces/Core/MSBuild.BuildHost/Rpc/Contracts/TextReaderExtensions.cs

```diff
@@ -19,7 +19,7 @@ internal static class TextReaderExtensions
         // If we're on .NET Core 7.0 the implementation is easy, but on older versions we don't have the helper and since we also don't have
         // Microsoft.VisualStudio.Threading's WithCancellation helper we have to inline the same approach.
 
-#if NET7_0_OR_GREATER
```

Comment:
Comment also needs updating.

Reply:
Although it's not clear to me why this isn't just `#if NET` now...

Reply:
> Although it's not clear to me why this isn't just `#if NET` now...

Sounds like a pre-existing issue. This PR should be just mechanical update - when we update our TFM, we move these `#if`s forward.